apps/full-site-editing/full-site-editing-plugin/full-site-editing/blocks/navigation-menu/index.js

```diff
@@ -0,0 +1,31 @@
+/**
+ * WordPress dependencies
+ */
+import { registerBlockType } from '@wordpress/blocks';
+import { __ } from '@wordpress/i18n';
+
+/**
+ * Internal dependencies
+ */
+import edit from './edit';
+import './style.scss';
+
+registerBlockType( 'a8c/navigation-menu', {
+	title: __( 'Navigation Menu' ),
+	description: __( 'Visual placeholder for site-wide navigation and menus.' ),
+	icon: 'layout',
+	category: 'layout',
```

Comment:
I'm not sure where we want to group this into 🤔 We can circle back in future PRs to see if we even want this in the inserter.

Reply:
Sure; layout seems fine for now. We could also make a separate "Full Site Editing" group for development too.